delivery_package_default_shipping_weight/models/stock_package_type.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools import float_compare
+
+
+class PackageType(models.Model):
+    _inherit = "stock.package.type"
+
+    package_default_shipping_weight = fields.Float(
```

Comment:
There is already a weight field on the package type. I would not an additional one but add a boolean 
```suggestion
    is_base_weight_shipping_weight = fields.Boolean(
    string="Is shipping weight",
    help="When flagged, the package type weight will be the package total weight. The product weight won't be added"
    )
```